source/solution_deploy/lib/standard_params-construct.ts

```diff
@@ -22,27 +23,43 @@ export class StandardParams extends Construct {
       {
         shortname: 'AFSBP',
         longname: 'aws-foundational-security-best-practices',
+        version: '1.0.0',
       },
       {
         shortname: 'CIS',
         longname: 'cis-aws-foundations-benchmark',
+        version: '1.2.0',
       },
       {
         shortname: 'PCI',
         longname: 'pci-dss',
+        version: '3.2.1',
       },
       {
         shortname: 'SC',
         longname: 'security-control',
+        version: '2.0.0',
       },
     ];
 
+    const CIS14Standard: Standard = {
+      shortname: 'CIS',
+      longname: 'cis-aws-foundations-benchmark',
+      version: '1.4.0',
+    };
+
     for (const standard of standards) {
       new StringParameter(this, `${standard.shortname}ShortName`, {
```

Comment:
Is it worth moving these back to their respective playbook stacks? I'm thinking it is, but let me know what you think.

This update works because the paths don't collide, but technically what's happening is CFN is creating these as new params as part of the top-level admin stack, then deleting the old ones. But we could also move them back to the playbook stacks so CFN updates the same resource instead.

I can't think of anything specific that would go wrong doing it the way it is here, but it splits up the parameters for a standard (shortname, status, remaps) between the playbook stack and the top-level stack. And if we release like this, we can't move it back to the playbook stack in a future release for the same reason - we'd be creating different resources with colliding paths. For that reason alone I'd lean toward moving it back to the stack it was in in 1.5.1.

You'll need to move it back to where it was [here](https://github.com/aws-solutions/aws-security-hub-automated-response-and-remediation/blob/main/source/lib/sharrplaybook-construct.ts#L54), but SC uses a different class, so you'll need to add it [there](https://github.com/aws-solutions/aws-security-hub-automated-response-and-remediation/blob/develop/source/playbooks/SC/lib/security_controls_playbook-construct.ts#L23) as well.

Reply:
What was the reasoning behind moving these in the first place? Looking at the history, these changes were introduced in a CR adding CIS 1.4 but didn't do anything to solve the problem (from my POV) I think it always made sense to have them in the runbook stacks, but don't have context of the original change.

Reply:
The original problem was that when the parameters are in the playbook stacks, the paths for CIS 1.2 and 1.4 would collide, since there's no version. So the solution was not deployable with both those playbooks enabled. I moved them here before I know that would cause updating the stack to fail.